.gitignore

```diff
@@ -82,3 +82,5 @@ sklearn/utils/_seq_dataset.pxd
 sklearn/utils/_weight_vector.pyx
 sklearn/utils/_weight_vector.pxd
 sklearn/linear_model/_sag_fast.pyx
+# virtualenv from advanced installation guide
+sklearn-env/
```

Comment:
This is not required anymore as long as you sync with `main`. We added this to the `.gitignore` file in another PR. Here is how you sync with upstream:

Assuming you have `upstream` setup:

```bash
git remote add upstream git@github.com:scikit-learn/scikit-learn.git
```

### Sync with upstream

```bash
git fetch upstream
git merge upstream/main
```

Reply:
done sir